arangod/VocBase/vocbase.cpp

```diff
@@ -1733,7 +1733,7 @@ void TRI_vocbase_t::releaseCollection(
 /// using a cid of > 0 is supported to import dumps from other servers etc.
 /// but the functionality is not advertised
 std::shared_ptr<arangodb::LogicalView> TRI_vocbase_t::createView(
-    arangodb::velocypack::Slice parameters) {
+    arangodb::velocypack::Slice parameters, bool isUserRequest) {
```

Comment:
Can we avoid passing this flag to vocbase?

Reply:
Not really
```
namespace storage_helper {

Result construct(LogicalView::ptr& view, TRI_vocbase_t& vocbase,
                 velocypack::Slice definition, bool isUserRequest) noexcept {
  return safeCall([&]() -> Result {
    TRI_set_errno(TRI_ERROR_NO_ERROR);  // reset before calling createView(...)
    auto impl = vocbase.createView(definition, isUserRequest);
```